tests/adapter/mongo/test_query.py

```diff
@@ -858,28 +861,39 @@ def test_get_overlapping_variant(real_variant_database, case_obj, variant_objs):
     adapter.load_variants(
         case_obj, variant_type="clinical", category="sv", rank_threshold=-10, build="37"
     )
-    # GIVEN a SV variant from this database
-    sv_variant = adapter.variant_collection.find_one({"category": "sv"})
+    # GIVEN a SV variant in this database
+    sv_variant = adapter.variant_collection.find_one({"_id": sv_variant_obj["_id"]})
     assert sv_variant
     sv_variant_id = sv_variant["_id"]
 
-    # WITH a given gene from sv variant
-    gene_id = sv_variant["hgnc_ids"][0]
+    # WITH a given gene on the SV
+    gene_id = 17978
+    updated_sv_variant = adapter.variant_collection.find_one_and_update(
+        {"_id": sv_variant["_id"]},
+        {"$set": {"hgnc_ids": [gene_id]}},
+        return_document=ReturnDocument.AFTER,
+    )
+
+    # Retrieve a SNV variant occurring in the same case:
+    snv_variant = adapter.variant_collection.find_one({"_id": variant_obj["_id"]})
+    assert snv_variant
 
-    # Retrieve a SNV variant occurring in the same gene:
-    snv_variant = adapter.variant_collection.find_one({"category": "snv"})
     # And arbitrary set its hgnc_ids to gene_id
     updated_snv_variant = adapter.variant_collection.find_one_and_update(
-        {"_id": snv_variant["_id"]}, {"$set": {"hgnc_ids": [gene_id]}}
+        {"_id": snv_variant["_id"]},
+        {"$set": {"hgnc_ids": [gene_id]}},
+        return_document=ReturnDocument.AFTER,
     )
+
     # THEN the function that finds overlapping variants to the snv_variant
```

Comment:
```suggestion
    # THEN the function that finds overlapping variants to the snv_variant should return the SV variant
```

Reply:
I dunno, it's there two lines below, but it should definitely be in definite form - thank you!